dexs/defx/index.ts

```diff
@@ -0,0 +1,38 @@
+import type { SimpleAdapter } from "../../adapters/types";
+import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
+import { CHAIN } from "../../helpers/chains";
+import { httpGet } from "../../utils/fetchURL";
+
+const URL = "https://api.defx.com/v1/open/analytics/market/overview";
+
+interface Response {
+  data:{
+    totalVol?: string;
+    dayVol?: string;
+  }
+}
+
+const fetch = async (timestamp: number) => {
+  const response: Response = (await httpGet(URL));
+  const dayTimestamp = getUniqStartOfTodayTimestamp(new Date(timestamp * 1000));
+  
+  const returnObj = {
+    totalVolume: response?.data.totalVol?.toString(),
+    dailyVolume: response?.data.dayVol?.toString(),
+    timestamp: dayTimestamp,
+  };
+  
+  return returnObj;
+};
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    "defx": {
```

Comment:
which chain you need list ?

Reply:
We are currently bridging from Arbitrum One to our own L1.

Mainnet: https://app.defx.com
Testnet: https://testnet.defx.com

Reply:
Please let us know if we should change in any other place  on account of being a (currently) private L1.

Reply:
should set this with `arbitrum` ?